### ISSUE TITLE
joined model found: Costa et al 2020

### ISSUE THREAD

**Issue by schuelet** [RakipInitiative/ModelRepository#317]
Is this joined model anything we want to keep? It seems quite old, but official.

I put it here knime://external-KNIME-Server/FSKX_Testing/internal_resources/689.fskx

**Comment by mfilter** [comment]
No, this was just a test file to show that the models created in MicroHibro and exported there as FSKX files can be read into FSK-Lab.
